File: urionlinejudge/1020 - Age in Days.java
// Created by islam elhady on 1/22/18.

import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner s = new Scanner(System.in);
           int days , value;
           
           days  = s.nextInt();
           
           System.out.printf("%d ano(s)\n", days/365);
           value = (days%365);
           System.out.printf("%d mes(es)\n",value/30);
           value = (value%30);
           System.out.printf("%d dia(s)\n", value);
 
    }
 
}

File: urionlinejudge/1020 - Age in Days.cpp
// Created by islam elhady on 9/5/20.

#include <iostream>
 
using namespace std;
 
int main() {
 
    int n ,ano, mes,dia,d;
    cin>>n;


    ano=n/365;
    dia=n%365;
    mes=dia/30;
    d =dia%30;

    cout<<ano<<" ano(s)"<<endl;
    cout<<mes<<" mes(es)"<<endl;
    cout<<d<<" dia(s)"<<endl;


return 0;
}